homeassistant/components/fibaro/scene.py

```diff
@@ -54,6 +55,13 @@ def __init__(self, fibaro_scene: SceneModel) -> None:
             identifiers={(DOMAIN, controller.hub_serial)}
         )
 
+        _LOGGER.debug(
```

Comment:
Please remove the logging or move it into the setup above. We don't want __init__ to have side effects

Reply:
Removed the logging, it isn't so important.